WooCommerce/Classes/ViewRelated/Products/ProductSelector/ProductRowViewModel.swift

```diff
@@ -140,6 +140,24 @@ final class ProductRowViewModel: ObservableObject, Identifiable {
         return currencyFormatter.formatAmount(priceAfterDiscount) ?? ""
     }
 
+    /// Formatted price label based on a product's price and quantity. Accounting for discounts, if any.
+    /// e.g: If price is $5, quantity is 10, and discount is $1, outputs "$49.00"
+    ///
+    var totalPriceAfterDiscountLabel: String? {
+        guard let price = price else {
+            return nil
+        }
+        guard let priceDecimal = currencyFormatter.convertToDecimal(price) else {
+            return nil
+        }
+        let quantityDecimal = NSDecimalNumber(decimal: quantity)
+        let subtotalDecimal = priceDecimal.multiplying(by: quantityDecimal)
+        let totalPriceAfterDiscount = subtotalDecimal.subtracting((discount ?? Decimal.zero) as NSDecimalNumber)
+
+        return currencyFormatter.formatAmount(totalPriceAfterDiscount) ?? ""
+
```

Comment:
```suggestion
        guard let price = price,
              let priceDecimal = currencyFormatter.convertToDecimal(price) else {
            return nil
        }
        let subtotalDecimal = priceDecimal.multiplying(by: quantity as NSDecimalNumber)
        let totalPriceAfterDiscount = subtotalDecimal.subtracting((discount ?? Decimal.zero) as NSDecimalNumber)

        return currencyFormatter.formatAmount(totalPriceAfterDiscount)
```

Optional, just tightens it up a little. Changes: 
- A single guard statement as the `else` is identical.
- Uses the cast to NSDecimalNumber for `quantity`, rather than making a new variable.
- Avoids returning an empty string given the optional signature.

Reply:
Thanks for the tips, changed here: https://github.com/woocommerce/woocommerce-ios/pull/11016/commits/6c7c559611624e7cfbdb2fcadf808b2f5c1e369a